src/nvim/api/vim.c

```diff
@@ -1468,3 +1473,19 @@ Float nvim__id_float(Float flt)
 {
   return flt;
 }
+
+Array nvim_list_uis(void)
+  FUNC_API_SINCE(1)
+{
+  Array alluis = ARRAY_DICT_INIT;
+  for (unsigned int i = 0; i < ui_count ; i++) {
+    Dictionary dic = ARRAY_DICT_INIT;
+    PUT(dic, "width", INTEGER_OBJ(uis[i]->width));
+    PUT(dic, "height", INTEGER_OBJ(uis[i]->height));
```

Comment:
also "rgb", `BOOLEAN_OBJ(uis[i]->rgb)`

Reply:
Yes, I'll list all the other properties now.